Symfony/CS/Tokenizer/Tokens.php

```diff
@@ -307,7 +307,7 @@ public function clearEmptyTokens()
         $limit = $this->count();
         $index = 0;
```

Comment:
I think it would be better to just remove this line and keep the assignment in the for block?